src/xcode/ENA/ENA/Source/Services/CCLService/CCLService.swift

```diff
@@ -319,12 +356,19 @@ class CCLService: CCLServable {
 					completion(.failure(.custom(customError)))
 				} else {
 					Log.error("Unhandled error \(error.localizedDescription)", log: .vaccination)
-					completion(.failure(.custom(DCCDownloadRulesError.RULE_CLIENT_ERROR(.boosterNotification))))
+					switch resourceType.ruleType {
+					case .boosterNotification:
+						completion(.failure(.custom(DCCDownloadRulesError.RULE_CLIENT_ERROR(.boosterNotification))))
+					case .invalidation:
+						completion(.failure(.custom(DCCDownloadRulesError.RULE_CLIENT_ERROR(.boosterNotification))))
```

Comment:
Is the type '.boosterNotification' correct?
Can't you just pass 'resourceType.ruleType' into 'RULE_CLIENT_ERROR(...)'?